docs/03-github/02-activation.mdx

```diff
@@ -37,20 +37,24 @@ Create a file called `.github/workflows/activation.yml` and add the below workfl
 
 ```yaml
 name: Acquire activation file
+
 on:
-  workflow_dispatch: {}
+  workflow_dispatch:
+
 jobs:
   activation:
     name: Request manual activation file 🔑
     runs-on: ubuntu-latest
     steps:
-      # Request manual activation file
       - name: Request manual activation file
         id: getManualLicenseFile
         uses: game-ci/unity-request-activation-file@v2
-      # Upload artifact (Unity_v20XX.X.XXXX.alf)
+        with:
+          unityVersion: 202x.x.xfx
+          # Include your unity version
```

Comment:
Did you find a use-case where you needed the license to be for a specific version? As far as I know this should work with the default (which is `'2019.2.11f1'`) for all versions.

If not; do you still think we should make this change? i.e. will it make it more understandable for others. If yes, we'll merge it.

Reply:
I will check a few thing tomorrow to not force something useless but I had a big struggle with the original example as it was not showing in my Actions in repo.
I tried a lot of thing and finally managed to show it with this structure.

Reply:
Let me make sure and I will get back on that

Reply:
So it was just my mistake as I just started with github action.
So I will close it then and btw great job this amazing project!

Reply:
Thanks for double checking and for the intention to help us improve GameCI :)